### PR TITLE
My First Pull Request !

### PR DESCRIPTION
**Pull request by zmekonnen251**
index.html, style.css files, and linter for local and remote repository added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 ## Author
 
-- GitHub: [@githubhandle](https://github.com/zmekonnen251)
-- Twitter: [@twitterhandle](https://twitter.com/mek_zela)
+- GitHub: [@zmekonnen251](https://github.com/zmekonnen251)
+- Twitter: [@mek_zela](https://twitter.com/mek_zela)
 - LinkedIn: [LinkedIn](www.linkedin.com/in/zelalem-g-mekonnen)
```
Comment (VanessaAoki) [inline README.md]: - [ ] [OPTIONAL]: Right now when I click on this icon it redirects me to a 404 page. To make your repo looks professional, you should double-check your links to make sure they all work! Also, consider changing the `[LinkedIn]` to your name, considering it already says that the following link will be your LinkedIn profile!

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @zmekonnen251, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] Your README file looks very professional and the PR has a descriptive title :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 

## Required Changes ♻️
- [ ] One of the project requirements was to add linters to your project, you can check them out [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js). _You should consider adding local linters in the gitignore file, to keep your project clean!_
- [ ] Your Pull Request should have a short summary of the implemented changes in the commentary. You should consider editing your comment to add that, you can read more about it [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [ ] According to the project requirements, your HTML and CSS file should contain the following:

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (zmekonnen251) [reply README.md]: Thank you Vanessa Aoki very your comment and I have corrected it.

Comment (zmekonnen251) [reply README.md]: Thank you @VanessaAoki for your helpful suggestion. I modified it as you suggested.

**Comment by zmekonnen251** [comment]
Hello @VanessaAoki, as you suggested to me I have corrected all mistakes that I made earlier. I added a "Hello Microverse" heading in the HTML file and I styled it in the CSS file. I added the linter too in the local and remote repository.

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
```
Comment (elmar8287) [inline .github/workflows/linters.yml]: - [ ] Please, consider that you need to set up a linter only for HTML and CSS. Please, refer this [link](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml)
Screen Shot 2022-01-04 at 23 22 58

File: style.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: red;
+  text-align: center;
+}
```
Comment (elmar8287) [inline style.css]: - [ ] Please make sure that you are using the class with styles for your `h1` instead of targeting it directly over the tag name.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @zmekonnen251 , 👋🏼  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (zmekonnen251) [reply style.css]: Thank you for your suggestion.

File: .github/workflows/linters.yml (hunk 2 above)
Comment (zmekonnen251) [reply .github/workflows/linters.yml]: Thank you @elmar8287  for your suggestion.

**Comment by zmekonnen251** [comment]
I have corrected all comments that I received from my code reviewers. I used HTML class to style an element instead of directly targeting the element itself. And another change that I made is, I previously used JavaScript linter but I removed the JavaScript linter now.

**Review by mmsesay — APPROVED** [review]
Hi @zmekonnen251 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zmekonnen251** [comment]
Thank you very much. All of you helped me a lot to know the GitHub flow and PR.

**Pull request merged**